tests/protocol/test_wire.py

```diff
@@ -566,6 +565,106 @@ def test_report_event_large_event(self, patch_send_event, *args):
         self.assertEqual(patch_send_event.call_count, 0)
 
 
+class TestWireClient(AgentTestCase):
+
+    def test_save_or_update_goal_state_it_should_save_new_goal_state_file(self):
```

Comment:
NIT: typo in name - `test_save_or_update_goal_state_should_save_new_goal_state_file` instead of `test_save_or_update_goal_state_it_should_save_new_goal_state_file`

Reply:
updated